discovery-infra/start_discovery.py

```diff
@@ -22,6 +23,18 @@
 import bootstrap_in_place as ibip
 
 
+# Any change here should be reflected in scripts/haproxy.cfg as well
+IMAGE_REGISTRIES = [
+    "quay.io",
```

Comment:
Since we are hard coding these, maybe we can have a single place that they are defined?

Reply:
and will be nice to have it in consts and not here